packages/core/utils/src/openInBrowser.js

```diff
@@ -49,7 +49,7 @@ function getAppName(appName: string): string {
 export default async function openInBrowser(url: string, browser: string) {
   try {
     const options =
-      typeof browser === 'string' ? {app: [getAppName(browser)]} : undefined;
+      typeof browser === 'string' && browser.length > 0 ? {app: [getAppName(browser)]} : undefined;
```

Comment:
```suggestion
      typeof browser === 'string' && browser.length > 0
        ? {app: [getAppName(browser)]}
        : undefined;
```
This should calm down prettier